weave/trace/op.py

```diff
@@ -208,6 +211,8 @@ def _create_call(func: Op, *args: Any, **kwargs: Any) -> "Call":
     if "api_key" in inputs_with_defaults:
         inputs_with_defaults["api_key"] = "REDACTED"
 
+    call_time_display_name = wvkw.get("display_name", None) if wvkw else None
```

Comment:
`get` missing case is implicitly `None`.

Reply:
done